kubernetes/helm_charts/core/inbound/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+  namespace: {{ .Values.namespace }}
+  annotations:
+    reloader.stakater.com/auto: "true"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+     rollingUpdate:
+        maxSurge: {{ .Values.strategy.maxsurge }}
+        maxUnavailable: {{ .Values.strategy.maxunavailable }}
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+    spec:
+{{- if .Values.imagepullsecrets }}
+      imagePullSecrets:
+      - name: {{ .Values.imagepullsecrets }}
+{{- end }}
+      volumes:
```

Comment:
@shiva-rakshith Do we need this?

Reply:
As of now we don't have and secrets, So we don't need this, removing it.

Reply:
What I meant is,  you have removed volumeMounts but still have retained volumes. Please remove volumes section and add the imagePullSecrets back. We need the image secrets. Please refer to the correct line of comment in review.

Reply:
Apologies for misunderstanding, i will make the correct changes.